tests/EnvironmentTest.php

```
@@ -24,124 +27,279 @@ public function testConstants(): void {
+  }
+
+  public function testNoPlatformWithCiSignalIsCi(): void {
+    self::envSet('CI', 'TRUE');
+
+    Environment::init(contextualize: FALSE, platforms: []);
+
+    $this->assertSame(Environment::CI, getenv('ENVIRONMENT_TYPE'));
+  }
+
+  public function testNoPlatformWithoutCiSignalIsLocal(): void {
+    Environment::init(contextualize: FALSE, platforms: []);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\EnvironmentDetector\Environment' in method 'testNoPlatformWithoutCiSignalIsLocal'. (undefined)

(StaticAccess)